fabric.md

```diff
@@ -10,30 +10,46 @@ _Hyperledger Fabric（以下简称Fabric）欢迎各位大神积极参与代码
 
 ## 安装开发环境
 * git客户端，通常需要您可以在命令行中执行git命令
+* [可选] git review插件,用于支持`git review`命令，安装方法可以参考 https://www.mediawiki.org/wiki/Gerrit/git-review
 * Go编程语言1.10.x以及配置好的开发环境，例如GOPATH，GOROOT以及相应的文件夹结构，请参考[Golang文档](https://golang.org/doc)
 * Docker 和 Docker Compose
-MacOSX, \*nix, Windows10需要安装Docker 17.06.2-ce或以上版本。Windows10以下版本的OS需要安装Docker Toolbox，同样要求17.06.2-ce或以上版本。
+MacOSX, Linix/Unix, Windows10需要安装Docker 17.06.2-ce或以上版本。Windows10以下版本的OS需要安装Docker Toolbox，同样要求17.06.2-ce或以上版本。
+* 安装libtool
+  * Ubuntu: `apt-get install -y libtool`
+  * MacOS: `brew install libtool`
 * MacOS的用户需要安装以下额外的工具
   * Xcode
   * 执行 `brew install gnu-tar --with-default-names` 安装gnutar来替换自带的bsdtar
-  * `brew install libtool`
-
-## 获取Linux Foundation ID (LFID)
-* 前往https://identity.linuxfoundation.org/ 注册一个LFID并登陆
-* 前往https://gerrit.hyperledger.org 并点击右上角Sign in，使用您的LFID登陆
-* 配置Gerrit使用ssh。可以参考[该章节](https://hyperledger-fabric.readthedocs.io/en/latest/Gerrit/lf-account.html#configuring-gerrit-to-use-ssh)
 
 ## 获取源码
-在$GOPATH/src/github.com/hyperledger下运行命令：
-```
-git clone ssh://LFID@gerrit.hyperledger.org:29418/fabric && scp -p -P 29418 LFID@gerrit.hyperledger.org:hooks/commit-msg fabric/.git/hooks/
-```
-该命令会克隆fabric源码，并下载githooks。
-
-下载源码后，在fabric根目录下，运行`make gotools`安装一些fabric开发依赖的go工具，例如golint
+1. 前往https://gerrit.hyperledger.org 并点击右上角Sign in，使用您的LFID(Linux Foundation ID)登陆
+    * 如果没有LFID（第一次使用），请前往https://identity.linuxfoundation.org/ 注册一个LFID
+2. 在项目列表找到fabric项目，进入项目主页
+    * `Projects` -> `List` -> `fabric`
+3. 点选左上方的`Clone with commit-msg hook`按钮
+    * *错误示例：如果没有登陆你的 LFID，你将看不到 `Clone with commit-msg hook`按钮，只能看到 clone | anonymous http 按钮*
+4. 在按钮同一栏的右边，选择验证方式 anonymous http|http|ssh，并复制其下方的命令
+    * ssh验证：配置gerrit使用ssh
+      - 参考[该章节](https://hyperledger-fabric.readthedocs.io/en/latest/Gerrit/lf-account.html#configuring-gerrit-to-use-ssh)
+      - 优点：操作时不用交互输入账户，密码
+      - 缺点：需要先生成本机的ssh pubkey文件，并在https://gerrit.hyperledger.org/的个人设置中将pubkey的内容加入列表中，才可以使用。每台开发机都需要做一次
+    * http验证 **推荐**：  
```

Comment:
glad to include `http` as an option! although I feel we shouldn't give preferences. let's simply list pro&con here, similar to what you did with `ssh`.